core/tests/protractor_utils/ClassroomPage.js

```diff
@@ -0,0 +1,40 @@
+// Copyright 2014 The Oppia Authors. All Rights Reserved.
```

Comment:
Year :)

Reply:
Done, maybe the first file with 2020 :).

Reply:
Looks like it, happy new year!

Reply:
You too!